explorer-server/code/address.js

```diff
@@ -156,13 +156,13 @@ const renderToken = (_value, _type, row) => {
   return '';
 };
 
-const updateLoading = (status) => {
+const updateLoading = (status, tableId) => {
```

Comment:
Good time to rename this function. Something like `updateTableLoading` or `updateTableLoadingSymbol` etc.

Also `status` should be `isLoading`